stl/inc/algorithm

```diff
@@ -7592,12 +8129,12 @@ namespace ranges {
     // clang-format off
     template <bidirectional_iterator _It, class _Pr, class _Pj>
         requires sortable<_It, _Pr, _Pj>
-    constexpr void _Insertion_sort_common(const _It _First, const _It _Last, _Pr _Pred, _Pj _Proj) {
+    constexpr _It _Insertion_sort_common(const _It _First, const _It _Last, _Pr _Pred, _Pj _Proj) {

```

Comment:
This is triggering my "It's weird to return an argument" senses.

Reply:
I mean this is a simple hyper optimization removing the need to increment `_First` in _Insertion_sort_isort_max_chunks